buildSrc/src/main/groovy/AwsMixinGenerator.groovy

```diff
@@ -247,25 +250,23 @@ class AwsMixinGenerator implements Plugin<Project> {
  */
 """
 
-  def mapperHeader = """\
+  def mapperConfigurerHeader = """\
 $licenseHeader
 package com.netflix.awsobjectmapper;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.MapperFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 
-public class AmazonObjectMapper extends ObjectMapper {
-  public AmazonObjectMapper() {
-    configure(MapperFeature.AUTO_DETECT_IS_GETTERS, false);
-    configure(SerializationFeature.INDENT_OUTPUT, true);
-    configure(SerializationFeature.WRITE_NULL_MAP_VALUES, false);
-    configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
-    configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    setPropertyNamingStrategy(new AmazonNamingStrategy());
+public class AmazonObjectMapperConfigurer {
+  public static void configure(ObjectMapper objectMapper) {
```

Comment:
I had thought of adding a convenience method in here but opted for the simpler approach. I could be convinced otherwise.

````java
public static ObjectMapper newMapper() {
  ObjectMapper mapper = new ObjectMapper();
  configure(mapper);
  return mapper;
}
````